driver/portfolio_runner.py

```diff
@@ -120,7 +121,7 @@ def run_sat(configs, executable, sas_file, plan_manager, final_config,
                 configs, pos, search_cost_type, heuristic_cost_type,
                 executable, sas_file, plan_manager, timeout, memory)
             if exitcode is None:
-                return
+                continue
```

Comment:
If one search has time limit 0 we want to switch to the next config, not abort the whole portfolio.

Reply:
The rationale behind the original code is that each portfolio should have a non-zero relative time limit, else it shouldn't be part of the portfolio. This means that the time limit hits 0 for all configs at the same time (when the overall time limit is exhausted). So I don't see the purpose of the change. Can you say a scenario where it makes sense?

Reply:
If config5 has a low relative time limit in the portfolio file, it might get an absolute time limit of 0. But there could be a config6 with a high relative time limit that gets a non-zero absolute time limit.

Reply:
How? I don't understand. Is there rounding involved that can round down to 0?

Reply:
I see now that indeed this is potentially rounded down before we get here. Then the change makes sense, sorry for the noise.